docker-compose.yml

```diff
@@ -94,6 +95,13 @@ services:
       - "ADMIN_KEY=${ADMIN_KEY}"
       - "ADMIN_ORG=${ADMIN_ORG}"
       - "GPG_PASSPHRASE=${GPG_PASSPHRASE}"
+      # MISP system settings
+      - "MEMORY_LIMIT=${MEMORY_LIMIT:-2048M}"
```

Comment:
Can all these variables be pre-fixed with PHP?
Also, default values should be set inside [core/files/entrypoint_fpm.sh](https://github.com/MISP/misp-docker/pull/101/files#diff-c07e7e7ca98bbc3fa5aee2ba87977b0e8741ec564d98a8d572e7f6ef46f11253) to keep it consistent with other files.